otherlibs/dune-rpc/private/exported_types.ml

```diff
@@ -34,6 +34,468 @@ module Loc = struct
   ;;
 end
 
+let sexp_pp (conv_tag : 'a Conv.value) : 'a Pp.t Conv.value =
```

Comment:
This generalises the previous sexp_pp function, and adds it to the API for use elsewhere.

Reply:
I actually had to introduce two versions of this function. The new version serialises tags as `(Tag (pair tag pp))` which is different to how it was done previously as `(Tag pp)`. Therefore I have included the old version too and made sure to version bump both diagnostics and the diagnostic poll.